database/mssql/scripts/dbscripts/PSP_PIMS_S19_00/Alter Down/01_PSP_PIMS_S19_00_Alter Down.sql

```diff
@@ -289,14 +289,6 @@ GO
 IF @@ERROR <> 0 SET NOEXEC ON
 GO
 
--- Drop index dbo.LSTERM_LEASE_TERM_STATUS_TYPE_CODE_IDX
```

Comment:
@dfilteau surprised to see changes to the IS19 alter down/alter up in this PR.

The problem with doing that is that we have already shipped IS19 and executed the alter up / alter down in the dev and test environments. If we now to try to run these new alter ups/alter downs on test, I'm not sure what the result will be, since the database does not match the state that these updated scripts are expecting.

I'll wait to here back on why these changes are here, but if we do have changes that need to be made to IS19 we'll need a plan to get those into dev and test.

Reply:
@dfilteau I just simulated this on my local by setting my local db to the previous version of IS19. I wasn't able to run any combination of alter down/alter up scripts to successfully go from the previous version of IS19 to this version. As a result, these changes may need to be rolled into IS20.

Reply:
or just strip out these changes for now, whatever is easier for you.